sympy/physics/continuum_mechanics/cross_section.py

```diff
@@ -0,0 +1,161 @@
+from sympy.geometry.polygon import Polygon, Point, Circle
+from sympy.core import sympify
+from sympy.printing import sstr
+from sympy.core import Symbol, S
+
+
+class CrossSection(object):
+    def __init__(self, shape, variable, *dimensions):
+        self.shape = shape
+        self.dimensions = dimensions
+        self.variable = variable
+
+        shape = shape.lower()
+        
+        if shape == "circular":
+            self._circular(self.dimensions)
```

Comment:
Probably these should be separate classes each with their own docstring that explains what arguments to provide for dimensions.